File: 5_1_2023/Main.java
import java.lang.*;
public class Main {

    public static int warOfNumbers(int[] arr) {
        int even = 0;
        int odd = 0;
        for (int val : arr) {
            if (val % 2 == 0)
                even += val;
            else
                odd += val;
        }
        return Math.abs(even - odd);
    }

    public static void main(String[] args) {
        int[] arr1 = {2, 8, 7, 5};
        int[] arr2 = {12, 90, 75};
        int[] arr3 = {5, 9, 45, 6, 2, 7, 34, 8, 6, 90, 5, 243};
        System.out.println(warOfNumbers(arr1));
        System.out.println(warOfNumbers(arr2));
        System.out.println(warOfNumbers(arr3));
    }
}

File: 5_1_2023/main.py
def sum_odd_and_even(arr):
    odd = 0
    even = 0
    for num in arr:
        if num % 2 == 0:
            even += num
        else:
            odd += num
    return [even, odd]

def main():
    arr1 = [1, 2, 3, 4, 5, 6]
    arr2 = [-1, -2, -3, -4, -5, -6]
    arr3 = [0, 0]
    print(sum_odd_and_even(arr1))
    print(sum_odd_and_even(arr2))
    print(sum_odd_and_even(arr3))

if __name__ == "__main__":
    main()
